validation/src/main/resources/application.conf

```diff
@@ -1,4 +1,24 @@
 hmda.validation.quality {
+  Q001 {
+    loan.amount=1000
+    loan.amount=${?Q001_LOAN_AMOUNT}
+    incomeMultiplier=5
+    incomeMultiplier=${?Q001_incomeMultiplier}
```

Comment:
@schbetsy Short answer, it could (i.e. yes) change. Some of these limits are tied to loan rules and others to cut-offs that seem like they would be quality issues so we could change them in the future.